packages/assistify-help-request/assets/stylesheets/helpRequest.less

```diff
@@ -47,3 +47,33 @@
 		}
 	}
 }
+
+.create-channel__inputs {
```

Comment:
@vickyokrm I renamed the file. The css does not only apply to the context anymore.
This help request context actually is not visualized anymore, but it should not be used as limiting class in the creation dialog. 
I recognized that the style should be applied to all textareas on the creation screen, thus I changed the selector accordingly.

Reply:
Noted.